### DISCUSSION TITLE
Thetis v2.10.3.6 Development

### DISCUSSION THREAD

**Discussion by ramdor** [ramdor/Thetis#398] [General]
### Development Build for 2.10.3.6

This is an in-progress **development build** for 2.10.3.6. It might be taken down at any time, and/or will probably get replaced without warning. The titlebar will show **[dev_2]**. Please read the change log below.

## Please uninstall any previous version, before installing this version

**(note: uninstalling using add/remove programs, leaves behind your database and does not remove it. You don't need to do anything with this, other than taking a backup perhaps, as Thetis should update/use appropriately)**

# !! CW Warning !!

**UPDATE: The CW issue has been resolved for Protocol 2. It requires a firmware update and this modified version of Thetis. Please see the [ApacheLab community forums here](https://community.apache-labs.com/viewtopic.php?f=32&t=4972)**

If you use CW and you have different RX and TX antenna ports configured, it is likely that full RF power will be present briefly on antenna ports configured as RX only. This may lead to damage of any sensitive equipment attached to the RX port such as active antennas and the like. This situation can occur if you utilise the CWX form, serial keying/PTT, **and standard hardware keys** plugged into the radio. This issue is currently under investigation and has existed in Thetis for an extended period (years and years). If you are interested in following the progress, you can read more about it on [GitHub here](https://github.com/ramdor/Thetis/issues/228).

**Note: You can revisit these release notes by clicking the button in the setup form within the General -> H/W Select tab.**

Please report any issue over on [GitHub here](https://github.com/ramdor/Thetis/issues/new/choose).

## v2.10.3.6 Change Log - WIP dev 2
- [add] -help command line option
- [fix] vac mic audio source can now be recorded
- [fix] split TXing out of a ham band will no longer disable split when extended mode is used. Note: 'Prevent TX when on different band to RX' option will be used if enabled and TX band has been moved for bandpass/lpf purposes  (ie when tx'ing on 11m)
- [fix] output power on ANT ports configured as RX only should be resolved. Requires an update to your firmware. See the [ApacheLab community forums here](https://community.apache-labs.com/viewtopic.php?f=32&t=4972)
- [change] the dB shift setting for AGC auto noise floor compensation is now a true shift from NF. You will probably need to tweak your previous setting
- [change] the shutdown log is defaulted off. Use command line param -logshutdown to cause Thetis to create a shutdown_log.txt file if you are having shutdown issues which can then be sent in for analysis

<hr /><em>This discussion was created from the release <a href='https://github.com/ramdor/Thetis/releases/tag/v2.10.3.6-dev_2'>Thetis v2.10.3.6 Development</a>.</em>

**Comment by Roturbo** [comment]
Hi,

Anan 100D

i have install this version to test the new firmware, one test version was found to work better than the others,
but after some time i start to notice that PS was not working correct with some mirror images.

I revert back to previous firmware and the problem was solved, than i start to notice power output 
at some point was much less than selected on drive slider,, for example if i chose 20w i only get 10,
or some value close, i change drive to max and the best was half of the power.

I was using a calibration to max 50w to match my PA input, sometimes i change to other calibrations
and it was the same problem, other times i start to have the correct power again, on the moment i have
less power i get some crazy spikes in power that make the PS extend the signal more than 40khz up
and down from my center frequency (SSB).

I have reset the database and the power was there 100w as normal or any other set in slider, so i start 
to think it was not hardware problem, so i go back to version 10.3.4 and all was working as before,
constant power and PS working without spikes.

Maybe is a DB problem or some other setting, i may start with a reset DB, but there are so many settings
and details to tune that it will take a lot of time.

Any test or suggestion ?

Regards

**Comment by Roturbo** [comment]
Hi again,, 

I have reverted to version 10.3.4  and the problem was gone, but after some days of usage i get the 
same exact problem again, using the 50w calibration on about 25w and one moment was ok the other
my power go to 10 or 12w max, on the same moment i load other PA calibration with 100w and the 
power was again normal and is working like this last 2 days.

Maybe is some type of combination that make the problem or change some value making the power
behave like a different scale.

I have a friend with a 200D that sometimes start to have PS problems after turn one the Diversity,
this is also a intermittent situation, on my case the version 10.3.4  takes longer to reveal the problem.

I´m going to test the latest version again to see if with 100w PA calibration it works better.

Regards.
